source_test.yaml

```
@@ -413,3 +413,17 @@
   repo_username: 'git'
   strict_validation: True
 
+- name: 'V8'
+  versions_from_repo: True
+  type: 0
+  ignore_patterns: [ '^(?!V8-advisory).*$' ]
+  repo_url: 'https://github.com/google/chromium-policy-vulnfeed.git'
+  detect_cherrypicks: False
+  extension: '.json'
+  db_prefix: ['V8-']
+  ignore_git: False
+  human_link: 'https://github.com/google/chromium-policy-vulnfeed/blob/main/advisories/V8-advisory.json'
```

```suggestion
  human_link: 'https://github.com/google/chromium-policy-vulnfeed/blob/main/advisories/'
  
```
To align with our validator for sourcerepositories, this link should point at the collection of advisories rather than a single one (regardless if there is only 1 advisory) :)